deepdiff/deephash.py

```diff
@@ -190,6 +191,7 @@ def __init__(self,
         self.ignore_private_variables = ignore_private_variables
         self.encodings = encodings
         self.ignore_encoding_errors = ignore_encoding_errors
+        self.ignore_list_order = ignore_list_order
```

Comment:
Please rename this to `ignore_iterable_order` to keep it consistent with the rest of parameter names in DeepDiff.